persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/GenePanelMapper.xml

```diff
@@ -296,7 +298,6 @@
     </insert>
 
     <insert id="insertGenePanelSampleProfileMap" parameterType="map" useGeneratedKeys="false">
-        insert into gene_panel_profile_map (SAMPLE_ID, PROFILE_ID, PANEL_ID)
-            values(#{sampleId}, #{profileId}, #{panelId})
+        update sample_profile set PANEL_ID = #{panelId} where SAMPLE_ID = #{sampleId} and GENETIC_PROFILE_ID = #{profileId}
```

Comment:
Hopefully in the import process we first create all the sample_profile entries, before we attempt to load the gene panel cases and update those existing records.